Common/cpp/SharedItems/Shareables.h

```diff
@@ -9,12 +9,7 @@
 #include "JSRuntimeHelper.h"
 #include "ReanimatedRuntime.h"
 #include "RuntimeManager.h"
-
-#ifdef __APPLE__
-#include <RNReanimated/Scheduler.h>
-#else
-#include "Scheduler.h"
-#endif
+#include "UIScheduler.h"
```

Comment:
Why was this __APPLE__ recurring in numerous places?

Reply:
Because there's also `Scheduler.h` file in react-native ([here](https://github.com/facebook/react-native/blob/main/packages/react-native/ReactCommon/react/renderer/scheduler/Scheduler.h)) which for some reason would take precedence over ours. The solution was to pass the full path to the header but it works only on iOS because on Android we need to use `#include "..."`.